istioctl/pkg/writer/pilot/status.go

```diff
@@ -92,8 +92,15 @@ func statusPrintln(w io.Writer, status *writerStatus) error {
 	listenerSynced := xdsStatus(status.ListenerSent, status.ListenerAcked)
 	routeSynced := xdsStatus(status.RouteSent, status.RouteAcked)
 	endpointSynced := xdsStatus(status.EndpointSent, status.EndpointAcked)
+	version := status.IstioVersion
+	if version == "" {
+		// If we can't find an Istio version (talking to a 1.1 pilot), fallback to the proxy version
+		// This is misleading, as the proxy version isn't always the same as the Istio version,
+		// but it is better than not providing any information.
+		version = status.ProxyVersion
```

Comment:
Would it make more sense to set version to `"Proxy " + status.ProxyVersion` to distinguish this when 1.2 _istioctl_ talks to 1.1 Istio?

Reply:
It would at least be more clear for people familiar with it although for someone who doesn't I think adding proxy doesn't really help. Aside from adding a long message about proxy vs Istio version I wasn't sure how we could convey that.

Reply:
It might be possible to report both proxy version and Istio version, and have the Istio version marked as "Unknown" when talking to Istio < 1.2.  I think that is too much information for most people.  I would just use `"*"` or `"Istio "` to indicate something imprecise is happening when 1.2 client talks to < 1.1 control plane.